spring-grpc-spring-boot-autoconfigure/src/main/java/org/springframework/grpc/autoconfigure/client/GrpcClientProperties.java

```diff
@@ -150,12 +150,11 @@ public Health getHealth() {
 			return this.health;
 		}
 
-		private final Map<String, ?> serviceConfig = new HashMap<>();
-
 		/**
-		 * The service config to use for the channel.
-		 * @return the service config
+		 * Map representation of the service config to use for the channel
```

Comment:
The config props generator was unable to pickup the docs from the getter - moved it to the field instead.